pages.fr/common/grep.md

```diff
@@ -18,7 +18,7 @@
 
 - Utilise des expressions rationnelles étendues (supporte `?`, `+`, `{}`, `()` et `|`) :
 
-`grep -E {{^regex$}} {{chemin/vers/fichier}}`
+`grep -E {{^expression_régulière$}} {{chemin/vers/fichier}}`
```

Comment:
Please remove the `^...$` parts here aswel.

Reply:
I think you reviewed on a older commit. All of your reviews are either outdated or the `^...$` isn't there.

Reply:
@Waples I think you commented on an out-of-date version of the page.  GitHub says "Outdated" at the top of this comment and I can't find `^expression_régulière$` in the Files Changed tab 🤔